sdk/keyvault/keyvault-keys/test/crypto.test.ts

```diff
@@ -9,7 +9,7 @@ import { ClientSecretCredential } from "@azure/identity";
 import { CryptographyClient, Key, KeysClient } from "../src";
 import { authenticate } from "./utils/testAuthentication";
 import TestClient from "./utils/testClient";
-import { isRecording } from "./utils/recorder";
+import { isRecording } from "./utils/recorderUtils";
```

Comment:
We already have `isRecordMode` and `isPlaybackMode` in the `@azure/test-utils-recorder`, but we are not exporting them in the index.ts file.
We can export them from there itself.

Reply:
Let's make an issue and follow up separately! 🌞

Reply:
Let's use this issue to track this work: https://github.com/Azure/azure-sdk-for-js/issues/4872